internal/controller/policymanifest_controller.go

```diff
@@ -0,0 +1,133 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	exceptionRecommender "github.com/giantswarm/exception-recommender/api/v1alpha1"
+	"github.com/go-logr/logr"
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	kyvernov2beta1 "github.com/kyverno/kyverno/api/kyverno/v2beta1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	giantswarmPolicy "github.com/giantswarm/kyverno-policy-operator/api/v1alpha1"
+)
+
+// PolicyManifestReconciler reconciles a PolicyManifest object
+type PolicyManifestReconciler struct {
+	client.Client
+	Scheme               *runtime.Scheme
+	Log                  logr.Logger
+	DestinationNamespace string
+	Background           bool
+}
+
+//+kubebuilder:rbac:groups=giantswarm.io,resources=policymanifests,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=giantswarm.io,resources=policymanifests/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=giantswarm.io,resources=policymanifests/finalizers,verbs=update
+
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+// TODO(user): Modify the Reconcile function to compare the state specified by
+// the PolicyManifest object against the actual cluster state, and then
+// perform operations to make the cluster state reflect the state specified by
+// the user.
+//
+// For more details, check Reconcile and its Result here:
+// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.17.0/pkg/reconcile
+func (r *PolicyManifestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	_ = log.FromContext(ctx)
+
+	var polman exceptionRecommender.PolicyManifest
+
+	if err := r.Get(ctx, req.NamespacedName, &polman); err != nil {
+		// Error fetching the policy manifest
+
+		if apierrors.IsNotFound(err) {
+			return ctrl.Result{}, nil
+		}
+
+		log.Log.Error(err, "unable to fetch policy manifest")
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	// TODO(user): your logic here
+
+	// kpe stands for Kyverno Policy Exception
+	kpe := kyvernov2beta1.PolicyException{}
+	// Set kpe destination namespace.
+	kpe.Namespace = r.DestinationNamespace
+	// Set kpe name.
+	kpe.Name = fmt.Sprintf("gs-kpo-%s-exceptions", polman.ObjectMeta.Name)
+	// Set labels.
+	kpe.Labels = make(map[string]string)
+	kpe.Labels["app.kubernetes.io/managed-by"] = "kyverno-policy-operator"
+	kpe.Labels["policy.giantswarm.io/policy"] = polman.ObjectMeta.Labels["policy.giantswarm.io/policy"]
+
+	kpe.Spec.Background = &r.Background
```

Comment:
This may also be possible to do in a helper function since we do this for all the controllers too.